.github/workflows/sanitizer-tests.yaml

```diff
@@ -32,14 +32,14 @@ jobs:
       - name: Install apt packages
         uses: ./src/Fast-DDS/.github/actions/install-apt-packages
 
-      - name: Install GTest
-        uses: ./src/Fast-DDS/.github/actions/install-gtest-linux
```

Comment:
It seems we are not using this action anymore, may as well delete it

Reply:
Addressed in 1563fc7774d965e0f6f8591e8f156c9554448e7e